guides/source/testing.md

```diff
@@ -2366,29 +2544,183 @@ class ChatRelayJobTest < ActiveJob::TestCase
 end
 ```
 
+Running tests in Continuous Integration (CI)
+--------------------------------------------
+
+Continuous Integration (CI) is a development practice where changes are
+frequently integrated into the main codebase, and as such, are automatically
+tested before merge.
+
+To run all tests in a CI environment, there's just one command you need:
+
+```bash
+$ bin/rails test
+```
+
+If you are using [System Tests](#system-testing), `bin/rails test` will not run
+them, since they can be slow. To also run them, add another CI step that runs
+`bin/rails test:system`, or change your first step to `bin/rails test:all`,
+which runs all tests including system tests.
```

Comment:
This information is repeated further up in the guide. But maybe that's okay since it's relevant to CI I guess.